packages/itwinui-css/src/slider/slider.scss

```diff
@@ -38,9 +37,10 @@ $tick-height: $iui-baseline; // 12px
 
 @mixin iui-slider-rail {
   position: absolute;
-  width: 100%;
-  height: $track-height;
   background-color: var(--iui-color-background-border);
+  @media (forced-colors: active) {
+    background-color: CanvasText;
+  }
 }
 
 @mixin iui-slider-thumb {
```

Comment:
I don't know if this is important, but one small thing I noticed was that this property might be increasing the size of the thumb from `14x14` to `16x16`. I think this is part of [Leah's PR](https://github.com/iTwin/iTwinUI/pull/661) and might be a needed property, but just mentioning what I noticed. Because this property was not there when I working on the [vertical slider PR](https://github.com/iTwin/iTwinUI/pull/707)

Reply:
Oops, looks like it selected the wrong lines. I was referring to this property
![image](https://user-images.githubusercontent.com/45748283/184420793-112c7b67-2409-4356-85cc-a642fce057ea.png)

Reply:
What property?

Reply:
`box-sizing: content-box` on line 48. This was just a small thing I noticed, so I thought I'd just mention it.

Reply:
I was looking into this and didn't notice any difference between the size of the thumb in the [live version](https://itwin.github.io/iTwinUI/slider.html) and [current version](https://itwin.github.io/iTwinUI/718/slider.html). They are both 18x18 (16 + 1px border on both sides).

Reply:
Ohh, I meant when I removed the `box-sizing` property, I noticed that the size of the thumb went from `16x16` to `14x14`. When I was working on the [vertical slider PR](https://github.com/iTwin/iTwinUI/pull/707), this property was not there. I think it came from [Leah's PR](https://github.com/iTwin/iTwinUI/pull/661) done after I worked on mine.

[Screencast from 08-17-2022 01:12:32 AM.webm](https://user-images.githubusercontent.com/45748283/185040275-20d7fdd6-a4a0-41bf-b20b-5f05e8141ff3.webm)

Reply:
Well yeah, it's not meant to be 14x14 so `content-box` was added to restore the size back to 16x16.

Reply:
**[RESOLVED]** Ohh, great. Was just confirming